lib/ruby/stdlib/net/http.rb

```diff
@@ -633,10 +633,6 @@ class << HTTP
     # is required to use the proxy, and p_no_proxy hosts which do not
     # use the proxy.
     #
-    # In JRuby, this will default to the JSE proxy settings provided in the
-    # 'http.proxyHost' and 'http.proxyPort' Java system properties, if they
-    # are set, falling back on environment variables otherwise.
-    #
```

Comment:
This comment - is it to be removed or kept?

Reply:
Yeah I'm confused why this didn't get picked up when I updated our fork. I'm re-pushing without this and some other non-functional OpenSSL changes.